users/users.controller.js

```diff
@@ -8,8 +8,8 @@ router.get("/", async (req, res) => {
 });
 
 router.post("/", async (req, res) => {
-  const { username, email, firstname, lastname, age } = req.body;
-  const user = await addUser(username, email, firstname, lastname, age);
+  const { username, email, firstname, lastname, age: dateOfBirth } = req.body;
```

Comment:
I would just change every instance of age to dateOfBirth, rather than renaming during destructuring.

Reply:
Resolved